### DISCUSSION TITLE
Emulating clicks on the left/right side of the touchpad

### DISCUSSION THREAD

**Discussion by LeoMilian** [r57zone/DualShock4-emulator#69] [Q&A]
I can't find a way to emulate the touchpad left side click either using a keyboard or an Xbox controller
The default config for the keyboard shows `TOUCHPAD-LEFT=H` but apparently this is not what I'm looking for since this does not do anything for me
- Is it possible to emulate clicks on the left/right side of the touchpad right now? 
- Also, it would be nice to have the option to remap the Xbox buttons. In this particular case, I'd like to remap the `BACK` button to the `click on the left side of the touchpad` event

**Comment by r57zone** [comment]
You need to press H + Enter or K + Enter (Enter presses the touchpad). If this is needed for uncharted 3, then the Share button does it there (F12 it seems).
On the Xbox controller, you can hold the Back button and move the right stick left or right and press the stick. You can reassign buttons in the configuration file (but without emulating several actions at the same time).
Study the documentation in its entirety, there are answers to most questions, also take a look at the config.

**Reply by LeoMilian** [reply]
According to VSC View what you describe is working as intended.
![image_2023-03-14_13-15-04](https://user-images.githubusercontent.com/81646299/225036135-04142590-8dad-471d-8927-915c7d5fa2e6.png)


 But for me, it does not work when actually playing on PS Plus. Is there any way to set the same uncharted 3 option for all the games?

**Reply by r57zone** [reply]
Unfortunately, I don’t know what the reason is, it seems that in the latest versions of PS RemotePlay and PSPlus, the virtual DualShock 4 (Vigem) was generally broken or disabled
